File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.cpp
#include <string>
#include <vector>

using namespace std;

bool arr[100];

int solution(vector<int> numbers) {
    int answer = -1;
    
    int n=0;
    for(int i=0;i<numbers.size();i++){
        arr[numbers[i]]=true;
    }
    
    for(int i=1;i<=9;i++){
        if(!arr[i]){
            n+=i;
        }
    }
    
    return n;
}

File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.java
class Solution {
    public int solution(int[] numbers) {
        
        
        // 0 ~ 9 까지의 숫자의 일부가 들어간 배열 
        int []arr= new int[10];
        
        for(int n : numbers){
            arr[n]++;
        }
        
        int sum =0;
        
        for(int i=1;i<=9;i++){
            if(arr[i]==0)
                sum+=i;
        }
        
        return sum;
    }
}